rebar.config

```diff
@@ -1,24 +1,71 @@
-%% vim: set ts=4 sw=4 ft=erlang et:
+%% vim: set ts=2 sw=2 ft=erlang et:
+
+{overrides,
+ [{override, ex_uri,
+   [{plugins, [rebar3_abnfc_plugin]},
+    {provider_hooks,
+     [{pre,
+       [{compile, {abnfc, compile}},
+        {clean, {abnfc, clean}}]}]}
+   ]}
+ ]}.
+
 {erl_opts,
  [debug_info,
-  warnings_as_errors,
+  {parse_transform, lager_transform},
   {platform_define, "^[0-9]+", namespaced_types}]}.
 
 {deps,
- [{redis, "", {git, "git@git.herokai.com:erlang_redis_pool.git", "master"}}
- ,{mochiweb, "", {git, "git@git.herokai.com:mochiweb.git", "R16B01"}}
- ,{pagerduty, "", {git, "git@git.herokai.com:erlang_pagerduty.git", "master"}}
- ,{redgrid, "", {git, "git@git.herokai.com:redgrid.git", {tag, "v1.0.3"}}}
- ,{redo, "^1.1", {git, "git@git.herokai.com:redo.git", "7c7eaef4cd65271e2fc4ea88587e848407cf0762"}}
- ,{nsync, "", {git, "git@git.herokai.com:nsync.git", "89cfad0ae1b38f252ca6371140473b44426d100b"}}
- ,{cowboy, "", {git, "git@git.herokai.com:ninenines_cowboy.git", {tag, "1.0.1"}}}
- ,{quoted, "", {git, "git@git.herokai.com:quoted.git", "master"}}
- ,{gproc, "", {git, "git@git.herokai.com:gproc.git", "master"}}
- ,{ex_uri, "", {git, "git@git.herokai.com:ex_uri.git", "master"}}
- ,{heroku_crashdumps, "", {git, "git@git.herokai.com:heroku_crashdumps.git", {tag, "0.1.0"}}}
- ,{lager, "", {git, "git@git.herokai.com:lager.git", "2.0.0"}}
- ,{batchio, "", {git, "git@git.herokai.com:batchio.git", "1.0.0"}}
- ,{recon, "", {git, "git@git.herokai.com:recon.git", "master"}}
- ,{folsom, "", {git, "git@git.herokai.com:folsom.git", "0.8.2"}}
- ,{folsom_cowboy, "", {git, "git@git.herokai.com:folsom_cowboy.git", "51842c8e8a89707bdf072287703c721939d55827"}}
+  [{redis, {git, "git@git.herokai.com:erlang_redis_pool.git", {branch, "master"}}}
+  ,{mochiweb, {git, "git@git.herokai.com:mochiweb.git", {tag, "R16B01"}}}
+  ,{pagerduty, {git, "git@git.herokai.com:erlang_pagerduty.git", {branch, "master"}}}
```

Comment:
Just a question: a lot of repo deps point to `master` branch. Is `git.herokai.com` supposed to be a stable copy of these deps (manually updated by someone), or just a mirror?